manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -72,9 +77,27 @@ protected function getOptions(): array
      */
     protected function createStore(): Psr6Store
     {
+        $cacheDir = $this->cacheDir ?: $this->kernel->getCacheDir().'/http_cache';
+
+        if (PhpFilesAdapter::isSupported()) {
+            $itemsCache = new PhpFilesAdapter('', 0, $cacheDir);
+        } else {
+            $itemsCache = new FilesystemAdapter('', 0, $cacheDir);
+        }
+
+        $tagsCache = $itemsCache;
+
+        if (class_exists(\SQLite3::class)) {
```

Comment:
```suggestion
        if (class_exists('SQLite3', false)) {
```

This would save us an unnecessary autoloader call.